app/models/photosynthesis_measurement.rb

```diff
@@ -1,4 +1,42 @@
+require 'csv'
+
 class PhotosynthesisMeasurement < ActiveRecord::Base
   include BatchImport
+  
+  belongs_to :leaf_part
+
+  validates :leaf_part_id, presence: true
+
+  validates :filename, presence: true
```

Comment:
Nice to see lots of validation here. I would tend to align the second parameter to make it more readable, but that's of course down to you.